### ISSUE TITLE
Didnt get couple of achievements at roundend

### ISSUE THREAD

**Issue by Ilgradhr** [tgstation/tgstation#83037]
Reporting client version: 515.1636

## Issue Summary

Did a bartending spree for a bit more than two hours and at roundend it showed that "The bar served 104 customers and made 3340 credits". But i did not get even Centcom Grade: Shitty Service

Other thing is I did the Ordeal of Sisyphus. Didnt drop the boulder, had help with shuttle got it to roundend and didnt get it. also took it from starting mine thingie

## Round ID:
228294

**Comment by Livrah** [comment]
In Ordeal of Sisyphus you need to get a boulder to Centcomm tiles. it will be after the shuttle docked and won't show in roundend report.

**Comment by Ilgradhr** [comment]
> In Ordeal of Sisyphus you need to get a boulder to Centcomm tiles. it will be after the shuttle docked and won't show in roundend report.

Okay that one is on me got blown up before i could get out of the shuttle.
After sleeping on this issue I have a question. Do you have to do the Centcom Grade as a barkeeper/chef?

**Comment by Livrah** [comment]
> Do you have to do the Centcom Grade as a barkeeper/chef?
>
Yes. I did Exemplary Service today, it doesn't show in roundend report either.

**Comment by san7890** [comment]
It should show up in the roundend report (via https://[tgstation13.org/parsed-logs/terry/data/logs/2024/05/04/round-228296/round_end_data.html](https://tgstation13.org/parsed-logs/terry/data/logs/2024/05/04/round-228296/round_end_data.html))

```txt
Service Statistics:
The restaurant served 0 customers and made 0 credits.
The bar served 104 customers and made 3340 credits.
In total, they earned 3340 credits!
Centcom is satisfied with service's job today.
```
